src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -33,8 +34,11 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
```

Comment:
In this controller are 2 methods with userId. And we need to replace all userId with authentication

Reply:
I agree - refactor the second method as well